tests/system/provisioning/vars/credentials.yml

```diff
@@ -1,16 +1,5 @@
 ### Ansible connection vars
-ssh_username: wazuh
-ssh_password: T3stP4ssw0rd--
-key_pair_name: devops-dev
-
-ssh_key_path_private: ~/.ssh/devops-dev.pem
```

Comment:
This vars are used in `ansible_hosts.j2` template to configure the connection to the instances, e.g:

https://github.com/wazuh/wazuh-qa/blob/f5a28434824ee89be03e91b3e27ac25b9cf400d8/tests/system/provisioning/roles/aws/templates/ansible_hosts.j2#L57

Reply:
Hi!,
You have to set up on the Ansible call due to sensitiveness of such variables
An example command
```
ansible-playbook deploy_aws_stack.yml --private-key /path/to/private.key -u root -e aws_access_key=**** -e aws_secret_key=**** -e aws_region=us-east-1 -e hosts_classified_destination=/any/path/to/hosts_file -e ansible_ssh_config_destination=/any/path/to/hosts_config_file -e stack_name=THE-STACK-NAME -e ssh_key_path_private=/path/to/private.key
```

Regards.